module-4/metaanalyse/metaanalyse.py

```diff
@@ -70,12 +70,13 @@ def main():
 
     args = parser.parse_args()
 
+    if not args.file or not args.directory:
```

Comment:
must be `and` - otherwise if you specify `-d` but not `-f` you will result in an error.

Reply:
absolut xD Thx <3